PyTDF/backend/Executor.py

```diff
@@ -0,0 +1,33 @@
+class Executor(object):
+
+    def check_env(self):
+        """
+        Method to check the current 
+        execution environment from
+        PYTDF_ENV variable
+        
+        """
+        
+        if not env or (env == "local"):
+            return "local"
+        return env
+
+    def execute(self, generator):
+        """
+        Method to execute the 
+        event-loop from the root
+        node based on the execution
+        environment
+
+        """
+
+        import os
+        env = os.environ.get('PYTDF_ENV')
```

Comment:
It is fine that we do the choice of the backend based on a variable for now, but at some point we have to make a final decision on what mechanism we use. Other options:
- Different import per backend: `from PyTDF import RDataFrame` for local, `from PyTDF.spark import RDataFrame` for distributed with Spark.
- Additional method: `PyTDF.use('spark')`, default being local.

Reply:
Yup, certainly 👍